src/App.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen, cleanup } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import App from './App';
+import store from './redux/store/configureStore';
+
+afterEach(() => cleanup());
+
+test('renders countries', () => {
+  render(
+    <Provider store={store}>
+      <App />
+    </Provider>,
+  );
+  const linkElement = screen.getByText(/Today's Covid details/i);
+  expect(linkElement).toBeInTheDocument();
+});
+
+const APP = (
+  <Provider store={store}>
+    <App />
+  </Provider>
+);
+
+describe('App Renders test', () => {
+  it('Renders App', () => {
+    const app = renderer.create(APP).toJSON();
+    expect(app).toMatchSnapshot();
```

Comment:
- [ ] Some of your tests are failing please try to fix that 
<img width="1440" alt="Screen Shot 2022-07-28 at 11 40 15" src="https://user-images.githubusercontent.com/61790587/181488346-7084f1f7-461d-4c25-9438-7e9b501f110f.png">